pkg/p2p/libp2p/libp2p_test.go

```diff
@@ -22,6 +22,10 @@ import (
 	"github.com/multiformats/go-multiaddr"
 )
 
+const (
+	DefaultWelcomeMessage = "Hello World!"
```

Comment:
Let's leave a default message blank (not set at all), and explicitly set it in tests if needed.